client/components/Events/ManageEventTemplatesModal.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-multi-comp */
+
+import {IFormGroup, IBaseRestApiResponse, IGenericListPageComponent} from 'superdesk-api';
+import {superdeskApi} from '../../superdeskApi';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal} from '../index';
+
+interface IProps {
+    handleHide(): void;
+}
+
+interface IEventTemplate extends IBaseRestApiResponse {
+    template_name: string;
+}
+
+export class ManageEventTemplatesModal extends React.PureComponent<IProps> {
+    static propTypes: any;
+
+    render() {
+        const {handleHide} = this.props;
+
+        const {getGenericListPageComponent, ListItemColumn, ListItem} = superdeskApi.components;
+        const {getFormFieldPreviewComponent, FormFieldType} = superdeskApi.forms;
+
+        const EventTemplatesComponent = getGenericListPageComponent<IEventTemplate>('events_template');
+
+        const {gettext} = superdeskApi.localization;
+
+        const nameField = {
+            label: gettext('Template name'),
+            type: FormFieldType.textSingleLine,
+            field: 'template_name',
+            required: true,
+        };
+
+        const formConfig: IFormGroup = {
+            direction: 'vertical',
+            type: 'inline',
+            form: [
+                nameField,
+            ],
+        };
+
+        const renderRow = (
+            key: string,
+            item: IEventTemplate,
+            page: IGenericListPageComponent<IEventTemplate>
+        ) => (
+            <ListItem
+                key={key}
+            >
+                <ListItemColumn ellipsisAndGrow noBorder>
+                    {getFormFieldPreviewComponent(item, nameField)}
+                </ListItemColumn>
+                <ListItemColumn noBorder>
+                    <div style={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        alignItems: 'center',
+                    }}>
+                        <button onClick={() => page.startEditing(item._id)}>
+                            <i className="icon-pencil" />
+                        </button>
+                        <button onClick={() => page.deleteItem(item)}>
```

Comment:
There are no notifications to the user when an item is deleted. Please refer to agenda management modal behavior.